mj_envs/envs/biomechanics/__init__.py

```diff
@@ -127,7 +127,7 @@
                 'viz_site_targets': ('THtip','IFtip','MFtip','RFtip','LFtip'),
                 'target_jnt_value': np.array([0, 0, 0, -0.0904, 0.0824475, -0.681555, -0.514888, 0, -0.013964, -0.0458132, 0, 0.67553, -0.020944, 0.76979, 0.65982, 0, 0, 0, 0, 0.479155, -0.099484, 0.95831, 0]),
                 'normalize_act': False,
-                'reset_type': "random",        # none, init, random
+                'reset_type': "none",        # none, init, random
```

Comment:
Technically this change is not needed, but given that env name is `HandPoseAMuscleFixed-v0` it seemed appropriate to make `reset_type` NOT `random`

Reply:
`reset_type` details --
`none` -- no reset (it will continue from where last rollout left off)
`init` -- reset to poses specified in init_qpos, init_qvel
`random` -- reset to a random pose
For `Fixed` env perhaps `init` might be a better choice.

Reply:
Fixed